lib/screens/authentication/verification/verification_screen.dart

```diff
@@ -29,16 +31,7 @@ class VerificationScreen extends StatelessWidget {
                 final pageCommand = state.pageCommand;
                 BlocProvider.of<VerificationBloc>(context).add(const ClearVerificationPageCommand());
                 if (pageCommand is PasscodeNotMatch) {
-                  ScaffoldMessenger.of(context).showSnackBar(
-                    SnackBar(
-                      backgroundColor: AppColors.canopy,
-                      content: Text(
-                        'Pincode does not match'.i18n,
-                        textAlign: TextAlign.center,
-                        style: Theme.of(context).textTheme.subtitle2,
-                      ),
-                    ),
-                  );
+                  eventBus.fire(ShowSnackBar.success('Pincode does not match'.i18n));
```

Comment:
@gguijarro-c-chwy Here I noticed a **great disadvantage** of the localized texts, which is that **they require the context**.

I thought of launching the bus events from the blocs or mappers and thus removing the listeners in the UI.
But it will not be possible because the texts have to be in the UI, otherwise there will be a mess passing the context in bloc events or whatever in fact I think Sky has started using static metodos for that reason ..

Reply:
I see
context should never reach the blocs, and should only live in the UI